src/index.js

```diff
@@ -213,13 +216,13 @@ class KeycloakAdapter {
         return stores;
     }
 
-    obtainGrantFromCode(code, redirectUri) {
-        const req = { 
-            session: { auth_redirect_uri: redirectUri } 
+    obtainGrantFromCode(code, redirectUri, sessionId) {
+        const req = {
+            session: {auth_redirect_uri: redirectUri}
         };
-        return this.grantManager.obtainFromCode(req, code);
+        return this.grantManager.obtainFromCode(req, code, sessionId);
```

Comment:
I guess it would be nice to add `sessionHost`  as the 4th parameter as well to leave no room for error.
Then we can handle both custom logout path (other than `k_logout`) as well as rewriting hapi's base path (path prefix) right here, instead in Keycloak configuration.